app/policies/planning_application_policy.rb

```diff
@@ -7,6 +7,26 @@ class PlanningApplicationPolicy < ApplicationPolicy
   alias_method :validate_step?, :editor?
   alias_method :archive?, :editor?
 
+  def show?
+    super || signed_in_viewer?
+  end
+
+  def index?
+    super || signed_in_viewer?
+  end
+
+  def edit?
+    super || signed_in_editor?
+  end
+
+  def update?
+    super || signed_in_editor?
+  end
+
+  def confirm_new?
+    signed_in_editor?
+  end
+
```

Comment:
you can replace these lines with
```ruby
alias_method :confirm_new?, :editor?
```

Reply:
Updated in https://github.com/unboxed/bops/pull/107/commits/fbe89c4d938e9250e78c9658809d4c03aa344484